proto/axelar/nexus/v1beta1/proposal.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+package axelar.nexus.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "axelar/nexus/exported/v1beta1/types.proto";
+
+option go_package = "github.com/axelarnetwork/axelar-core/x/nexus/types";
+option (gogoproto.goproto_getters_all) = false;
+
+// CallContractsProposal is a gov Content type for calling contracts on other
+// chains
+message CallContractsProposal {
+  option (gogoproto.goproto_stringer) = false;
+
+  string title = 1;
+  string description = 2;
+  repeated ContractCall contract_calls = 3 [ (gogoproto.nullable) = false ];
+}
+
+message ContractCall {
```

Comment:
```suggestion
message CallContract {
```
We generally use ContractCall for the events, once the calls have been made. So more consistent this way? Not a strong opinion

Reply:
I find it weird when the message is not a noun. CallContract in the gateway makes sense though cuz it's a method and should be a verb.